read-fonts/src/tables/os2.rs

```diff
@@ -0,0 +1,8 @@
+//! The [os2](https://docs.microsoft.com/en-us/typography/opentype/spec/os2) table
+
+use font_types::Tag;
+
+/// 'os/2'
+pub const TAG: Tag = Tag::new(b"OS/2");
+
+include!("../../generated/generated_os2.rs");
```

Comment:
Add a simple test. https://github.com/googlefonts/fontations/pull/63/files#diff-d6837cecda6aef91f3f5b2bc2f5da22cc55c7d8778d1f270e0ebfaa2abf7d822 exhibits the simplest way I'm currently aware of. Indeed you could use that same binary font and read it's OS/2.